### ISSUE TITLE
Add automated URL Caching for in-house URLs In Addition to Manual Entries

### ISSUE THREAD

**Issue by hollowaykeanho** [ZORALab/Hestia#95]
# Description
```
Please provide a short description of what feature you're looking forward to
enhance below. Please include the story behind your idea as well to give a
better visualization of your idea.
```
Since HestiaLIBS in HestiaHUGO is very stable now, we can add an automated URL caching feature for PWA offline list in addition to the manual entries.

Only applies to in-house URLs (baseURL).

# Expected Behavior
```
Please specify the expected behavior of your requested enhancement. Some great
and helpful pointers are your expected interface (e.g. command patterns,
simple sketches of the user interface, etc).
```
Processed in-house URLs are automatically added to cached list.

# Current Behavior
```
Please specify the current behavior (e.g. workaround, blockage, etc).
```
in-house URLs have to be manually added to the cached list.

# Attachment
```
Please drag and drop the necessary data files (e.g. screenshot, logs, etc)
below.
```

**Comment by hollowaykeanho** [comment]
Implemented in 0d4a0690cb9a5373b5f750bc69815a36164afa65

**Comment by hollowaykeanho** [comment]
Will not implement this feature due to Hugo's poor memory management with `.Store`. It exihibits a "memory leak" phenomemon despite the memory were manually deleted and renewed. 

Not much of a choice. Implementation removed in 547d23e874ee5c78edcda0d6da4f814279f8cf86.
